Dashboard.Storage/Azure/BuildUtil.cs

```diff
@@ -71,6 +71,30 @@ public List<BuildFailureEntity> GetTestCaseFailures(DateTimeOffset startDate, st
             return _buildFailureDateTable.ExecuteQuery(query).ToList();
         }
 
+        public List<string> GetViewNames(DateTimeOffset startDate)
+        {
+            var filter = FilterUtil
+                .SinceDate(ColumnNames.PartitionKey, startDate)
+                .And(FilterUtil.Column(nameof(BuildResultEntity.ViewName), null, ColumnOperator.NotEqual));
+            var query = new TableQuery<BuildResultEntity>()
```

Comment:
See an example here:  

https://github.com/jaredpar/jenkins/blob/efc1e9861110cd80e93a75672d8f71e3cd56f8fa/ApiFun/Program.cs#L245

Essentially just use `DynamicTableEntity` as the entity type and then specify the exact columns you want in the `Select` query.  In this case it would be just `ViewName`